serialise/bench/versus/Macro/DeepSeq.hs

```diff
@@ -1,9 +1,13 @@
+{-# LANGUAGE CPP #-}
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 module Macro.DeepSeq where
 
 import Macro.Types
 
 import Control.DeepSeq
+#if MIN_VERSION_base(4,9,0)
```

Comment:
shouldn't this rather be `MIN_VERSION_deepseq`? the `rnf1`/`rnf2` availability is not a proper function of the `base` version